### PR TITLE
Improve saving project feedback

### PR DESCRIPTION
**Pull request by ClementPasteau**
https://user-images.githubusercontent.com/4895034/202217169-55afbd01-4670-47ae-afbe-e7cc7ed14229.mov

### CODE REVIEW

File: newIDE/app/src/ProjectsStorage/CloudStorageProvider/CloudProjectWriter.js [hunk 1]
```diff
@@ -118,14 +118,23 @@ export const getWriteErrorMessage = (
   return t`An error occurred when saving the project, please verify your internet connection or try again later.`;
 };
 
-export const generateOnChooseSaveProjectAsLocation = (
+export const generateOnChooseSaveProjectAsLocation = ({
+  authenticatedUser,
+  setDialog,
+  closeDialog,
+}: {
```
Comment (4ian) [inline newIDE/app/src/ProjectsStorage/CloudStorageProvider/CloudProjectWriter.js:125]: maybe use {| |}

File: newIDE/app/src/MainFrame/index.js [hunk 2]
```diff
@@ -207,6 +208,7 @@ export type State = {|
   editorTabs: EditorTabsState,
   snackMessage: string,
```
Comment (4ian) [inline newIDE/app/src/MainFrame/index.js:209]: You could try a `key` on the Snackbar to see if this component can animate when the message changes.

File: newIDE/app/src/ProjectsStorage/index.js [hunk 3]
```diff
@@ -52,10 +52,11 @@ export type StorageProviderOperations = {|
     wasSaved: boolean,
     fileMetadata: FileMetadata,
   |}>,
-  onChooseSaveProjectAsLocation?: (
+  onChooseSaveProjectAsLocation?: ({
```
Comment (4ian) [inline newIDE/app/src/ProjectsStorage/index.js:55]: {||}

File: newIDE/app/src/MainFrame/index.js (hunk 2 above)
Comment (ClementPasteau) [reply newIDE/app/src/MainFrame/index.js:209]: I went for a different strategy, using a different method which closes the snackbar and reopens it after a short delay.
It looks pretty nice (see video)

**Pull request merged**